lib/assert.h

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Why is this file called assert.h?
I think that this file is unneeded, and should be removed.
In any case, it has to be renamed.